opentelemetry/instrumentation/otel-collector-config.yaml

```diff
@@ -0,0 +1,68 @@
+receivers:
+  otlp:
+    protocols:
+      http:
+        endpoint: "0.0.0.0:4318"
+  filelog:
```

Comment:
one downside of using the filelog receiver, and relying on it for translation (e.g. parsing trace-related fields), is that users running on managed services (cloud run, GKE, GCE) who copy the application logic won't get the expected behavior.  We could, instead, make the example send the gcp-specific fields (e.g. trace with project prefix), but that would go against the goal of not including gcp-specific behavior.

Reply:
Is it okay if we leave it this way for simplicity, and have a larger discussion within the team to find the best way forward on managed runtimes?

cc @aabmass @jsuereth

Reply:
I'm definitely in-favor of leaving it this way for simplicity.

Reply:
SGTM